src/services/cherry-picker.ts

```diff
@@ -156,6 +114,24 @@ export class CherryPicker {
     return node
   }
 
+  // Fetch app/node's service or failure logs from redis
+  async fetchRawLogs(blockchain: string, rawNodeIDs: string[], logType: string): Promise<{ [id: string]: string }> {
+    const rawServiceLogs = {} as { [id: string]: string }
+
+    const redisKeys = rawNodeIDs.map(function (rawNodeID) {
+      return `${blockchain}-${rawNodeID}-${logType}`
+    })
+
+    const rawRedisLogs = await this.redis.mget(redisKeys)
+
+    let logCount = 0
+    rawNodeIDs.forEach(async (rawNodeID) => {
```

Comment:
I think we don't need the `async` here
```suggestion
    rawNodeIDs.forEach(rawNodeID => {
```